magento/pwa-studio
# PWA-1551: Slider cypress test

## Description

Add Cypress Coverage for Pagebuilder Slider 

## Related Issue

Closes PWA-1551

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps

- Cypress divider.spec.js should pass
- Steps -
- Go to `venia-integration-test dir`
- Run `yarn install`
- Run `yarn test`
- Run `slider.spec.js ` from cypress ui


#### Test scenario(s) for direct fix/feature





#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations





#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist



-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.
